apps/lwip/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2019, Data61
+# Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+# ABN 41 687 119 230.
+#
+# This software may be distributed and modified according to the terms of
+# the BSD 2-Clause license. Note that NO WARRANTY is provided.
+# See "LICENSE_BSD2.txt" for details.
+#
+# @TAG(DATA61_BSD)
```

Comment:
This is obsolete, please use `SPDX-License-Identifier: BSD-2-Clause`.

Reply:
In this case (because it is "ours") the rest of the header should be collapsed as well into just the copyright line and SPDX identifier like in the rest of the sources.